HIP/hip-412.md

```diff
@@ -360,67 +522,106 @@ The following is the formal definition of this schema using JSON Schema notation
 For more info see here: https://json-schema.org/
 ```json
 {
-    "title": "Token Metadata",
-    "type": "object",
-    "properties": {
-        "name": {
-            "type": "string",
-            "description": "Identifies the asset to which this token represents."
-        },
-        "description": {
-            "type": "string",
-            "description": "Describes the asset to which this token represents."
-        },
-        "image": {
-            "type": "string",
-            "description": "A URI pointing to a resource. Conditionally Optional."
-        },
-        "type": {
-            "type": "string",
-            "description": "Mime type of image. Required if image is defined."
-        },
-        "files": {
-            "type": "object",
-            "description": "Array of files.",
-            "properties": {
-                "uri": {
-                    "type": "string",
-                    "description": "A URI pointing to a resource."
-                },
-                "type": {
-                    "type": "string",
-                    "description": "Mime type of the resource."
-                },
-                "metadata_uri": {
-                    "type": "string",
-                    "description": "uri pointing to the metadata for this file"
-                }
-            },
-        },
-        "format": {
-            "type": "string",
-            "description": "Name of the format used by the NFT."
-        },
-        "properties": {
-            "type": "object",
-            "description": "Optional. Arbitrary properties. Values may be strings, numbers, object or arrays."
-        },
-        "localization": {
-            "type": "object",
-            "description": "Array of localized metadata.",
-            "properties": {
-                "uri": {
-                    "type": "string",
-                    "description": "A URI pointing to the localized metadata json file for this locale."
-                },
-                "locale": {
-                    "type": "string",
-                    "description": "A two-letter locale."
-                }
-            }
-        },            
-    },
-    "required": [ "name" ]
+	"title": "Token Metadata",
+	"type": "object",
+	"properties": {
+		"name": {
+			"type": "string",
+			"description": "Identifies the asset to which this token represents."
+		},
+		"creator": {
+			"type": "string",
+			"description": "Identifies the artist name(s)."
+		},
+		"creatorDID": {
+			"type": "string",
+			"format": "uri",
+			"description": "Points to a decentralized identifier to identify the creator.",
+		},
+		"description": {
+			"type": "string",
+			"description": "Describes the asset to which this token represents."
+		},
+		"image": {
+			"type": "string",
+			"format": "uri",
+			"description": "A URI pointing to a resource. Conditionally Optional."
```

Comment:
As recommended in [EIP-1155](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1155.md#erc-1155-metadata-uri-json-schema) which originally comes from [Instagram](https://help.instagram.com/1631821640426723)
```suggestion
			"description": "A URI pointing to a resource with mime type image/* representing the asset to which this token represents. Consider making any images at a width between 320 and 1080 pixels and aspect ratio between 1.91:1 and 4:5 inclusive. Conditionally Optional."
```